tests/unit_tests/test_wave_06.py

```diff
@@ -110,7 +115,20 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # Assert
+    assert result == True
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+
+    assert item_a in tai.inventory
+    assert item_b in tai.inventory
+    assert item_f in tai.inventory
+
+    assert item_d in jesse.inventory
+    assert item_e in jesse.inventory
+    assert item_c in jesse.inventory
```

Comment:
👍 This is a thorough check for all of the inventory items.

Since this test results in a swap occurring, but we don't have a clear specification on where items should end up when swapped, a length check with an exhaustive set of `in` checks is the most flexible way to check that items are where they should be.